### DISCUSSION TITLE
Welcome to cloudmusic-vscode Discussions!

### DISCUSSION THREAD

**Discussion by YXL76** [YXL76/cloudmusic-vscode#182] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sdttttt** [comment]
![b07df97ef8a45407f87d0ba0bb5822d7453025f9](https://user-images.githubusercontent.com/42728902/102155792-37245f80-3eb7-11eb-9e3a-dbc797b39691.jpg)
